tools/accuracy_checker/accuracy_checker/preprocessor/crop.py

```diff
@@ -99,22 +104,25 @@ def process_data(data, dst_height, dst_width, central_fraction, use_pillow, is_s
             cropped_data = Image.fromarray(data).crop((j, i, j + new_width, i + new_height))
             return np.array(cropped_data)
 
-        if width < new_width or height < new_height:
-            resized = np.array([width, height])
-            if resized[0] < new_width:
-                resized = resized * new_width / resized[0]
-            if resized[1] < new_height:
-                resized = resized * new_height / resized[1]
-            data = cv2.resize(data, tuple(np.ceil(resized).astype(int)))
-
-        height, width = data.shape[:2]
-        start_height = (height - new_height) // 2
-        start_width = (width - new_width) // 2
-        if is_simple_case:
-            # support GeometricOperationMetadata array for simple case only -- without tiling, pyramids, etc
-            metadata.setdefault('geometric_operations', []).append(GeometricOperationMetadata('crop', {}))
-
-        return data[start_height:start_height + new_height, start_width:start_width + new_width]
+        if not no_resize:
```

Comment:
looks like if no_resize enabled, you do nothing, not only in case, when image small than crop size...
on my view, the check that no resize should be done inside branch  where we check height and width and change new_width or/and new_height to original width and height respectively in case when no_resize enabled

Reply:
Crop moved into CornerCrop class